config/main.py

```diff
@@ -33,6 +35,70 @@ def run_command(command, display_cmd=False, ignore_error=False):
     if proc.returncode != 0 and not ignore_error:
         sys.exit(proc.returncode)
 
+def interface_alias(interface_name):
+    """Return default interface name if alias name is given as argument
+    """
+
+    cmd = 'sonic-cfggen -d --var-json "PORT"'
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+
+    port_dict = json.loads(p.stdout.read())
+
+    if interface_name is not None:
+        for port_name in natsorted(port_dict.keys()):
+            if interface_name == port_dict[port_name]['alias']:
+                return port_name
+        print "Invalid interface {}".format(interface_name)
+        sys.exit(0)
+
+def interface_name_to_alias(interface_name):
+    """Return alias interface name if default name is given as argument
+    """
+
+    cmd = 'sonic-cfggen -d --var-json "PORT"'
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+
+    port_dict = json.loads(p.stdout.read())
+
+    if interface_name is not None:
+        for port_name in natsorted(port_dict.keys()):
+            if interface_name == port_name:
+                return port_dict[port_name]['alias']
+        print "Invalid interface {}".format(interface_name)
+        sys.exit(0)
+
+
+
+def set_interface_mode(mode):
+    """Modify IFMODE env variable in user .bashrc
+    """
+    sudo_user = os.getenv('SUDO_USER')
+    user = os.getenv('USER')
+    set_mode = "IFMODE={}".format(mode)
+
+    if not sudo_user:
+        bashrc = "/{}/.bashrc".format(user)
+    else:
+        bashrc = "/home/{}/.bashrc".format(sudo_user)
+
+    f = open(bashrc,'r')
+    filedata = f.read()
+    f.close()
+
+    if "IFMODE" not in filedata:
+        newdata = filedata + set_mode
```

Comment:
You should probably prepend a newline here to be safe. If the file happens to end with no newline, the `IFMODE` line will be concatenated onto the existing final line.